python/dolfinx/fem/function.py

```diff
@@ -608,12 +608,21 @@ def tabulate_dof_coordinates(self) -> npt.NDArray[np.float64]:
         return self._cpp_object.tabulate_dof_coordinates()
 
 
-def VectorFunctionSpace(mesh: Mesh, element: typing.Union[ElementMetaData, typing.Tuple[str, int]],
-                        dim=None) -> FunctionSpace:
+def VectorFunctionSpace(
+        mesh: Mesh,
```

Comment:
Formatting is a bit ugly. Can it be make to look nicer?

Reply:
Would this be better?
```python
def VectorFunctionSpace(mesh: Mesh,
                        element: typing.Union[basix.ufl_wrapper._BasixElementBase,
                                              ElementMetaData, typing.Tuple[str, int]],
                        dim=None) -> FunctionSpace:
```

Reply:
Yes, looks better.